source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift

```diff
@@ -6,18 +6,25 @@ protocol ACRChoiceButtonDelegate: NSObjectProtocol {
 
 class ACRChoiceButton: NSView, NSTextFieldDelegate, InputHandlingViewProtocol {
     weak var delegate: ACRChoiceButtonDelegate?
+    public var choiceSetButtonConfig: ChoiceSetButtonConfig = .default
     public var buttonValue: String?
-    public var buttonType: NSButton.ButtonType = .switch
     public var idString: String?
     public var valueOn: String?
     public var valueOff: String?
     
-    override init(frame frameRect: NSRect) {
-        super.init(frame: frameRect)
+    private let renderConfig: RenderConfig
+    private let isMultiSelect: Bool
+    
```

Comment:
```suggestion
    private let choiceSetType: ChoiceSetType 
    
```
You can create a new enum which has `radio` and `checkbox`.
`isMultiSelect` doesn't look good from a button perspective.